<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>340</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Weather</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sources.qrc">
    <normaloff>:/xml/scrpis/Humidity.png</normaloff>:/xml/scrpis/Humidity.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="CityLabel">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>42</y>
      <width>145</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>CITY</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Temperature">
    <property name="geometry">
     <rect>
      <x>338</x>
      <y>118</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>气温</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="sources.qrc">:/xml/scrpis/Temperature.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="AirQuality">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>118</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>空气质量</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="sources.qrc">:/xml/scrpis/AirQuality.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Humidity">
    <property name="geometry">
     <rect>
      <x>338</x>
      <y>42</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>湿度</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="sources.qrc">:/xml/scrpis/Humidity.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="UVRadition">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>200</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>紫外线指数</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="sources.qrc">:/xml/scrpis/UVRadiationIntensity.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cold">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>200</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>感冒指数</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="sources.qrc">:/xml/scrpis/Cold.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Wear">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>200</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>穿衣指数</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="sources.qrc">:/xml/scrpis/Wear.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="WashCar">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>200</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>洗车指数</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="sources.qrc">:/xml/scrpis/WashCar.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Sport">
    <property name="geometry">
     <rect>
      <x>338</x>
      <y>200</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>运动指数</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="sources.qrc">:/xml/scrpis/Sport.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="AirPolution">
    <property name="geometry">
     <rect>
      <x>414</x>
      <y>200</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>空气污染指数</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="sources.qrc">:/xml/scrpis/Air.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Wind">
    <property name="geometry">
     <rect>
      <x>187</x>
      <y>42</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>风向/风力</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="sources.qrc">:/xml/scrpis/Wind.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="WindLabel">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>42</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="TemperatureLabel">
    <property name="geometry">
     <rect>
      <x>414</x>
      <y>118</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="AirQualityLabel">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>118</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="HumidityLabel">
    <property name="geometry">
     <rect>
      <x>414</x>
      <y>42</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="UVRaditionText">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>276</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>UV</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ColdText">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>276</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>COLD</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="WearText">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>276</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Wear</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="WashCarText">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>276</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Wash car</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="SpostText">
    <property name="geometry">
     <rect>
      <x>338</x>
      <y>276</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Sport</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="AirPolitionText">
    <property name="geometry">
     <rect>
      <x>414</x>
      <y>276</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Air</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="TodayText">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>148</y>
      <width>74</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="indent">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="menuButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>menu</string>
    </property>
    <property name="toolTipDuration">
     <number>1</number>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="sources.qrc">
      <normaloff>:/xml/scrpis/menu.png</normaloff>:/xml/scrpis/menu.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="MoveBar">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="sources.qrc">:/xml/scrpis/move.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="TodayPic">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>148</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>1</number>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sources.qrc"/>
 </resources>
 <connections/>
</ui>
